docs/user-manual/xr/input-sources.md

```diff
@@ -16,7 +16,15 @@ for (let i = 0; i < inputSources.length; i++) {
 }
 ```
 
-Input sources can be added and removed dynamically. This can be done by connecting physical devices or by switching input devices by the underlying platform, and some input sources are transient - and have a lifespan only during their primary action (e.g. touch screen tap in an AR session on mobile). You can subscribe to `add` and `remove` events:
+Input sources can be added and removed dynamically. This can be done by connecting physical devices or by switching input devices by the underlying platform.
```

Comment:
```suggestion
Input sources can be added and removed dynamically. This can be done by connecting physical devices or by switching input devices via the underlying platform.
```